gdk4/src/content_formats.rs

```diff
@@ -1,7 +1,7 @@
 // Take a look at the license at the top of the repository in the LICENSE file.
 
 use crate::ContentFormats;
-use glib::translate::*;
+use std::fmt;
```

Comment:
any reason for removing the  `use glib::translate::*;`?

Reply:
I don't remember removing it, so no. Will add back in.